centreon/doc/API/centreon-cloud-api.yaml

```diff
@@ -21,7 +16,7 @@ info:
   license:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
-  version: "24.04"
+  version: "24.07"
```

Comment:
on peut virer ça?

Reply:
@nmagniezCentreon I don't know, it's the version entered on the url to access at the endpoint
`
servers:
  - url: '{protocol}://{server}:{port}/centreon/api/{version}'
`

@nfragnet Do you know if it's possible to point to 'latest' here, for example?